src/components/InformationCard.js

```diff
@@ -141,8 +139,7 @@ InformationCard.propTypes = {
   title: PropTypes.string,
   type: PropTypes.string,
   bleachingSubItems: PropTypes.object,
-  pieChartContent: PropTypes.array,
-  pieChartLegend: PropTypes.object,
+  pieChartContent: PropTypes.object,
```

Comment:
This might be an opportunity to check out Proptypes.shape

No sweat if you don't want to do it for this round.